test/activation.cpp

```diff
@@ -1,318 +0,0 @@
-/*******************************************************************************
- *
- * MIT License
- *
- * Copyright (c) 2017 Advanced Micro Devices, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in all
- * copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
- *
- *******************************************************************************/
-#include "test.hpp"
-#include <array>
-#include <iostream>
-#include <iterator>
-#include <limits>
-#include <memory>
-#include <miopen/activ.hpp>
-#include <miopen/miopen.h>
-#include <miopen/stringutils.hpp>
-#include <miopen/tensor.hpp>
-#include <utility>
-
-#include "driver.hpp"
-#include "get_handle.hpp"
-#include "tensor_holder.hpp"
-#include "verify.hpp"
-
-std::string to_name(miopenActivationMode_t m)
-{
-#define STRING_CASE(x) \
-    case x: return #x; break;
-    switch(m)
-    {
-        STRING_CASE(miopenActivationPASTHRU)
-        STRING_CASE(miopenActivationLOGISTIC)
-        STRING_CASE(miopenActivationTANH)
-        STRING_CASE(miopenActivationRELU)
-        STRING_CASE(miopenActivationSOFTRELU)
-        STRING_CASE(miopenActivationABS)
-        STRING_CASE(miopenActivationPOWER)
-        STRING_CASE(miopenActivationCLIPPEDRELU)
```

Comment:
...and this?

Reply:
@atamazov  the above code seems to be from old activation.cpp

Configs are spewed out by << operator as below

struct ActivationConfig
{
    size_t N;
    size_t C;
    size_t H;
    size_t W;
    friend std::ostream& operator<<(std::ostream& outStream, const ActivationConfig& activConfig)
    {
        return outStream << "N:" << activConfig.N << " C:" << activConfig.C
                         << " H:" << activConfig.H << " W:" << activConfig.W;
    }
};

Here's an example
 EXPECT_EQ(status, miopenStatusSuccess)
        << "Backward activation failed. Config: " << activ_config;

Reply:
> ...and this?
@atamazov the above code seems to be from old activation.cpp

Configs are spewed out by << operator as below
`struct ActivationConfig
{
size_t N;
size_t C;
size_t H;
size_t W;
friend std::ostream& operator<<(std::ostream& outStream, const ActivationConfig& activConfig)
{
return outStream << "N:" << activConfig.N << " C:" << activConfig.C
<< " H:" << activConfig.H << " W:" << activConfig.W;
}
};
`

Here's an example

`EXPECT_EQ(status, miopenStatusSuccess)
<< "Backward activation failed. Config: " << activ_config;`

Reply:
It prints dimensions of one tensor, but the activation primitive has more parameters.

Reply:
Not resolved. miopenActivationCLIPPEDRELU is missing. Please "unresolve" this review comment.

/cc @JehandadKhan